src/file.bash

```diff
@@ -1026,54 +990,27 @@ assert_no_sticky_bit() {
 assert_not_symlink_to() {
   local -r sourcefile="$1"
   local -r link="$2"
-  # If OS is linux
-  if [[ `uname` == "Linux" ]]; then
```

Comment:
This if cascade effectively disables the function on platforms other than macos/linux. Is this intended?

Reply:
I decided this is a defect and fixed it be defaulting to the linux implementation. That way you will at least get some error when running on a non supported system instead of a silent failure to test at all.